### ISSUE TITLE
file:// SKIA_BINARIES_URL

### ISSUE THREAD

**Issue by zdufour-asp** [rust-skia/rust-skia#685]
Hi y'all, thanks to whoever reads this :)

This comment says that `file://` is supported for `SKIA_BINARIES_URL`, but I'm getting an "unknown scheme" error when using `file://./skia-binaries.tar.gz` / `file://skia-binaries.tar.gz`

https://github.com/rust-skia/rust-skia/blob/9f9f03029eab651b708a2260c116ab7a8e0fe68c/skia-bindings/build_support/binary_cache/env.rs#L16-L19

Is there a simple way to enable this? I have 0 rust experience, so I'm not really capable of contributing a PR, but I'm guessing this might be where a conditional would need to be added:

https://github.com/rust-skia/rust-skia/blob/661a0bcb3a0bbb34969fb9829904759dbf5cfe81/skia-bindings/build_support/binary_cache/utils.rs#L5-L6

### workaround

A workaround is to use a file-server (e.g. `python -m http.server`):

1. Download the binary release from your platform (the `github.com/.../releases/skia-binaries...tar.gz` in your output, not the `codeload.github.com` file)
2. in a separate terminal window/tab, navigate to your downloads folder and `python -m http.server`
3. Wherever you were trying to do stuff earlier, 

```sh
SKIA_BINARIES_URL=http://localhost:8000/name-of-the-file-you-downloaded.tar.gz whatever-cargo-build-stuff-you-were-doing
```

**Comment by pragmatrix** [comment]
Hi, thank you for writing this up. There is already an untested PR #645 that implements that.

Also #644.

**Comment by pragmatrix** [comment]
`file://` support for `SKIA_BINARIES_URL` landed in master through #645, closing.
